frappe/core/doctype/report/test_report.py

```diff
@@ -282,3 +282,55 @@ def test_toggle_disabled(self):
 
 		# Set user back to administrator
 		frappe.set_user('Administrator')
+
+	def test_add_total_row_for_tree_reports(self):
+		report_settings = {
+			'tree': True,
+			'parent_field': 'parent_value'
+		}
+
+		columns = [
+			{
+				"fieldname": "parent_column",
+				"label": "Parent Column",
+				"fieldtype": "Data",
+				"width": 10
+			},
+			{
+				"fieldname": "column_1",
+				"label": "Column 1",
+				"fieldtype": "Float",
+				"width": 10
+			},
+			{
+				"fieldname": "column_2",
+				"label": "Column 2",
+				"fieldtype": "Float",
+				"width": 10
+			}
```

Comment:
Since we are changing how the total for percent fields are calculated a field type with percent should also be added